courses/serializers.py

```diff
@@ -149,3 +150,47 @@ class Meta:
             'requirements',
             'sections',
         ]
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = [
+            'email',
+            'phone'
+        ]
+
+
+class UserSerializer(serializers.ModelSerializer):
+    profile = UserProfileSerializer(read_only=False)
+
+    def update(self, instance, validated_data):
+        prof, _ = UserProfile.objects.get_or_create(user=instance)
```

Comment:
I could be completely wrong on this but I'm not sure this is 100% necessary? I think since this isn't `many=True`, Django REST Framework can figure out how to update the object.

Reply:
I tried it and it said you can't update nested models... is there any way to get around that besides overriding update?  I read on stackexchange that you have to override but SE django posts are often out of date...

Reply:
in that case then yeah this works